### PR TITLE
vendor c/common@main

### PR DESCRIPTION
**Pull request by vrothberg**
Adjust the capabilities e2e tests since some have recently been dropped.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by vrothberg** [comment]
@flouthoc @giuseppe @Luap99 PTAL

**Review by flouthoc — COMMENTED** [review]
Tests are blowing up.

File: test/e2e/inspect_test.go [hunk 1]
```diff
@@ -524,10 +524,12 @@ var _ = Describe("Podman inspect", func() {
 
 		data := inspect.InspectContainerToJSON()
 		Expect(data).To(HaveLen(1))
-		Expect(data[0].HostConfig.CapDrop).To(HaveLen(3))
+		Expect(data[0].HostConfig.CapDrop).To(HaveLen(4))
 		Expect(data[0].HostConfig.CapDrop[0]).To(Equal("CAP_AUDIT_WRITE"))
 		Expect(data[0].HostConfig.CapDrop[1]).To(Equal("CAP_MKNOD"))
 		Expect(data[0].HostConfig.CapDrop[2]).To(Equal("CAP_NET_RAW"))
+		// FIXME: Why is sys_chroot dropped?
+		Expect(data[0].HostConfig.CapDrop[3]).To(Equal("CAP_SYS_CHROOT"))
```
Comment (vrothberg) [inline test/e2e/inspect_test.go:532]: @rhatdan PTAL

Comment (vrothberg) [reply test/e2e/inspect_test.go:532]: A bug in computing the dropped caps?  Need to figure that out before 4.4.

Comment (Luap99) [reply test/e2e/inspect_test.go:532]: This was changed in https://github.com/containers/common/pull/1240

Comment (vrothberg) [reply test/e2e/inspect_test.go:532]: But why are they listed as dropped?  There are over 30 capabilities and other ones we don't enable by default aren't listed as dropped.

Comment (Luap99) [reply test/e2e/inspect_test.go:532]: Ah I see what you mean, looks like inspect uses not the container.conf default but rather the oci runtime spec default:
https://github.com/containers/podman/blob/34cc61d167fe429eb76383cf2a09701a9b2533dd/libpod/container_inspect_linux.go#L129-L177

**Comment by vrothberg** [comment]
I'll clean up tomorrow.  I am sure we can get inspect right, thanks @Luap99

**Comment by vrothberg** [comment]
It looks like NET_RAW is needed for some tests to pass using ping etc.  At that point, I think its best to revert the change in c/common to unblock ongoing work.

**Comment by vrothberg** [comment]
https://github.com/containers/common/pull/1245 reverts the caps changes.

**Pull request closed without merging**